cmd/s3manager/main.go

```diff
@@ -36,6 +38,12 @@ func main() {
 	if !ok {
 		port = "8080"
 	}
+	basepath, ok := os.LookupEnv("BASE_PATH")
+	if !ok {
+		basepath = ""
+	}else{
```

Comment:
This `else` is not necessary and should be omitted. To add the prefix slash, I would rather leave that up to the user and explicitly describe in the readme that it is expected.